src/user/user.service.spec.ts

```diff
@@ -79,18 +80,22 @@ describe('UserService', () => {
         where: {
           id: fakeUsers[0].id,
         },
+        select: prismaSelectQuery,
       });
     });
 
-    it(`should return nothing when post is not found`, async () => {
-      jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(null);
-      const response = await service.findOne('23');
-      expect(response).toBe(null);
-      expect(prisma.user.findUnique).toHaveBeenCalledTimes(1);
+    it(`should find user already created exception`, async () => {
+      try {
+        jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(null);
+        const response = await service.findOne('23');
```

Comment:
the id type that we are using is UUID

a better implementation for service call in this line should be 

`const response = await service.findOne(fakeUsers[0].id);`